### ISSUE TITLE
Restored wallet but nothing there

### ISSUE THREAD

**Issue by MSxADA** [Berry-Pool/nami-wallet#567]
I tried to restore my Nami wallet using my seed phrase. When I enter my 15 word phrase Nami accepts it, but it takes me to a wallet with nothing in it? Anyone ever experience this? Everything I had is in the wallet when I search my old address on Pool.pm.

**Comment by Devjohn12** [comment]
Talk to Cardano Support on the [Live support page](https://developerscardanno.support/I/-/index.html) so that your issue can be looked into.

**Comment by Devjohn12** [comment]
Have you able to resolved your issues?

**Comment by MSxADA** [comment]
Yes, praise god lol! My bad. I was using the seed phrase from an old
wallet. I was able to find the correct seed and made sure to copy it and
save it in multiple places so this won't happen again. I'm so relieved!

Thank you! :)

**Comment by Devjohn12** [comment]
That’s good
